libndt7/build.gradle

```diff
@@ -23,10 +27,6 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    api 'com.squareup.okhttp3:okhttp:3.11.0'
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    api 'com.squareup.okhttp3:okhttp:3.14.0'
```

Comment:
(Random: have you seen that okhttp3 is being rewritten in Kotlin?)

Reply:
Yeah, it's not really critical to those who use the library but the biggest win will probably be better Null / Nonnull as a result